pkg/apis/serving/v1alpha1/servingruntime_types.go

```diff
@@ -70,7 +70,7 @@ type ServingRuntimePodSpec struct {
 	// Cannot be updated.
 	// +patchMergeKey=name
 	// +patchStrategy=merge
-	Containers []Container `json:"containers" patchStrategy:"merge" patchMergeKey:"name" validate:"required"`
```

Comment:
@Suresh-Nakkeran the `Container` struct above (line 41) should also now be removed.

Reply:
Removed. Thanks!